imgui-binding/build.gradle

```diff
@@ -34,3 +35,19 @@ jar {
         attributes  'Automatic-Module-Name': 'imgui.binding'
     }
 }
+
+tasks.register('makePropertyResource') {
+    def directory = file "src/main/resources/META-INF"
```

Comment:
Referring to my review, path can look like: `file "$buildDir/makePropertyResource"`